cypress/Shared/LandingPage.ts

```diff
@@ -1,5 +1,10 @@
 export class LandingPage {
 
-    public static readonly logoutButton = '#main > :nth-child(2) > :nth-child(3)'
     public static readonly loginSuccessmsg = '#main > :nth-child(2) > div'
-}
\ No newline at end of file
+    public static readonly newMeasureButton = '[data-testid=create-new-measure-button]'
+    public static readonly createMeasureButton = '[data-testid=create-new-measure-save-button]'
+    public static readonly cancelButton = '[data-testid=create-new-measure-cancel-button]'
+    public static readonly enterMeasureName = '[data-testid=measure-name-text-field]'
```

Comment:
Would enter measure name a few others here be part of a new shared Class for New Measure page or something like that? https://dev-madie.hcqis.org/measure/create